modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -215,6 +215,9 @@ def prepare_appointment(appointment)
         appointment[:requested_periods] = nil if booked?(appointment) && VAOS::AppointmentsHelper.cerner?(appointment)
 
         convert_appointment_time(appointment)
+
+        appointment[:station], appointment[:ien] = *extract_station_and_ien(appointment)&.first(2)
```

Comment:
This slightly unusual syntax is in order to avoid rubocop yelling about the class being too many lines

Reply:
I might be misunderstanding whats happening here. Would `appointment[:station], appointment[:ien] = extract_station_and_ien(appointment)` be shorter and also work?

Reply:
I needed a way to guard against there not being 2 returns in the event that it's nil. Afaik, if `extract_station_and_ien(appointment)` returns nil, `appointment[:station], appointment[:ien] = extract_station_and_ien(appointment)` would fail. There's a chance `extract_station_and_ien` doesn't even return an array, just `nil`